content/en/docs/chart_best_practices/templates.md

```diff
@@ -31,62 +31,73 @@ For that reason, _all defined template names should be namespaced._
 Correct:
 
 ```yaml
-{{- define "nginx.fullname" }}
-{{/* ... */}}
-{{ end -}}
+{{- define "nginx.fullname" -}}
+# ...
+{{- end }}
 ```
 
 Incorrect:
 
 ```yaml
 {{- define "fullname" -}}
-{{/* ... */}}
-{{ end -}}
+# ...
+{{- end }}
 ```
+
 It is highly recommended that new charts are created via `helm create` command
 as the template names are automatically defined as per this best practice.
 
 ## Formatting Templates
 
 Templates should be indented using _two spaces_ (never tabs).
 
-Template directives should have whitespace after the opening  braces and before
+Template directives should have whitespace after the opening braces and before
 the closing braces:
 
 Correct:
-```
-{{ .foo }}
-{{ print "foo" }}
-{{- print "bar" -}}
-```
 
-Incorrect:
-```
-{{.foo}}
-{{print "foo"}}
-{{-print "bar"-}}
+```yaml
+{{- /* Conditionally create service account */}}
+{{- if .Values.serviceAccount.create }}
```

Comment:
This will error out if `serviceAccount` is a null value. I would refrain from using nested values in examples. Especially since the ONLY thing this example is trying to convey is "template formatting".

The original examples were just fine IMO. Why are we changing this?

Reply:
No more serviceAccount.create or nested values! Original example restored!